x-pack/legacy/plugins/siem/public/components/search_bar/selectors.ts

```diff
@@ -5,9 +5,8 @@
  */
 
 import { createSelector } from 'reselect';
```

Comment:
Wonder why @elastic/siem isn't auto added to reviewers 🤔

Reply:
I think they didn't define their code owners.
Think this would be a good idea!